concordium-consensus/tests/scheduler/SchedulerTests/SmartContracts/V1/Checkpointing.hs

```diff
@@ -750,3 +965,6 @@ tests =
                 checkpointingTest4 spv pvString
                 checkpointingTest5 spv pvString
                 checkpointingTest6 spv pvString
+                checkpointingTest7 spv pvString
+                checkpointingTest8 spv pvString
+                checkpointingTest9 spv pvString
```

Comment:
Shouldn't we specify that these will succeed in P6 but fail in P4/P5?

Reply:
We can. I did not think this was very important since the failure mode is so odd. But added now.